definitions/enums/region.yml

```diff
@@ -1,5 +1,6 @@
 ---
 name: Region
+description: All the regions a server can be in.
```

Comment:
Since a Region enum represents only a single region and not all, maybe "The name of a region that a guild's voice server can be located in."